README.adoc

```diff
@@ -334,12 +333,26 @@ exception if either of the method-level constraints is violated.
 [role='command']
 include::{common-includes}/devmode-build.adoc[]
 
-Navigate to the http://localhost:9080/openapi/ui[http://localhost:9080/openapi/ui^] URL. Expand the
-`/beanvalidation/validatespacecraft POST request to validate your spacecraft bean`
+// Static guide instruction
+ifndef::cloud-hosted[]
+Navigate to the http://localhost:9080/openapi/ui[http://localhost:9080/openapi/ui^] URL.
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Use the following command to get the URL.
+Open your browser and check out your service by going to the URL that the command returns.
+```
+echo http://${USERNAME}-9080.$(echo $TOOL_DOMAIN | sed 's/\.labs\./.proxy./g')/openapi/ui
+```
+{: codeblock} 
+endif::[]
+
+Expand the `/beanvalidation/validatespacecraft POST request to validate your spacecraft bean`
```

Comment:
Expand the `/beanvalidation/validatespacecraft POST request to validate your spacecraft bean` section 
--->
Expand the `/beanvalidation/validatespacecraft` POST request to validate your spacecraft bean section